admin/services/data-access/pupil-access-arrangements.data.service.js

```diff
@@ -17,9 +17,9 @@ pupilAccessArrangementsDataService.sqlFindPupilAccessArrangementsByPupilId = asy
   SELECT
   *, pfs.code AS pupilFontSizeCode, pcc.code AS pupilColourContrastCode
   FROM ${sqlService.adminSchema}.[pupilAccessArrangements] paa
-  LEFT JOIN ${sqlService.adminSchema}.[pupilFontSizes] pfs
+  FULL OUTER JOIN ${sqlService.adminSchema}.[pupilFontSizes] pfs
```

Comment:
Are you 100% sure you want full outer?
https://www.w3schools.com/sql/sql_join_full.asp
i would assume not, but please check.

Reply:
@GuyHarwood updated to LEFT OUTER as discussed